gstreamer-editing-services-sys/build.rs

```diff
@@ -0,0 +1,66 @@
+extern crate pkg_config;
+
+use pkg_config::{Config, Error};
+use std::env;
+use std::io::prelude::*;
+use std::io;
+use std::process;
+
+fn main() {
+    if let Err(s) = find() {
+        let _ = writeln!(io::stderr(), "{}", s);
+        process::exit(1);
+    }
+}
+
+fn find() -> Result<(), Error> {
+    let package_name = "gst-editing-services-1.0";
+    let shared_libs = ["ges-1.0"];
+    let version = if cfg!(feature = "v1_16") {
+        "1.16"
+    } else if cfg!(feature = "v1_8") {
```

Comment:
Here (and in `Cargo.toml`) I usually add a feature for every major version to keep things easier for users. Even if there was no new API in that major version. They can then simply select e.g. `v1_12` for all the GStreamer crates they care about instead of having to look at the exact versions for each single crate

Reply:
This was all generated, I need something special to enable that behaviour?